app/models/ticket_desk.rb

```diff
@@ -1,2 +1,5 @@
 class TicketDesk < ApplicationRecord
+  def self.online
+    find_by(connection: 'Online')
```

Comment:
```suggestion
    find_by!(connection: 'Online')
```
You treat it as an [invariant](https://en.wikipedia.org/wiki/Invariant_(mathematics)#Invariants_in_computer_science) in the system, so I think it would be better to raise an exception than silently fail later (this is a difference between `find_by` and `find_by!`).